portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -782,4 +841,59 @@ definition {
 		AssertElementNotPresent(locator1 = "Portlet#HEADER_DUPLICATE");
 	}
 
+	@description = "This test search app and view purchased apps in marketplace purchased page."
+	@priority = "3"
+	test ViewPurchasedApp {
+		task ("Given login marketplace store") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			AppManager.loginMarketplaceStore();
+
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Purchased");
+
+			AssertElementPresent(
+				locator1 = "UserBar#USER_SIGN_OUT",
+				value1 = "Sign Out");
+		}
+
+		task ("When go to marketplace store and search apps and purchase") {
```

Comment:
Update the description from _When go to marketplace store and search apps and purchase_ to _When purchase the Control Panel Documentation app in marketplace store_

Reply:
done